internal/app/secretless/handlers/pg/backend.go

```diff
@@ -52,6 +52,15 @@ func (h *Handler) ConfigureBackend() (err error) {
 	delete(values, "username")
 	delete(values, "password")
 
+	// TODO: remove because of hack
+
+	// TODO: what are options for, it's weird that any additonal
```

Comment:
maybe this should be a gh issue instead of a comment? I've had this thought before too, but it's going to get lost here